cms/cms_toolbars.py

```diff
@@ -421,30 +421,32 @@ def change_language_menu(self):
             remove = [(code, languages.get(code, code)) for code in self.page.get_languages() if code in languages]
             add = [l for l in languages.items() if l not in remove]
             copy = [(code, name) for code, name in languages.items() if code != self.current_lang and (code, name) in remove]
-            if add:
+
+            if add or remove or copy:
                 language_menu.add_break(ADD_PAGE_LANGUAGE_BREAK)
+
+            if add:
+                add_plugins_menu = language_menu.get_or_create_menu(LANGUAGE_MENU_IDENTIFIER+'-add', _('Add Translation'))
```

Comment:
Better use ``format`` here instead of mere concatenation (same for the other menus)